octoprint_mrbeam/static/js/material_settings.js

```diff
@@ -295,6 +320,13 @@ $(function () {
                     "Watch out for dedicated laser plywood - it has better surface quality and better glue."
                 ),
                 safety_notes: "",
+                type: "product",
+                url: gettext("https://www.mr-beam.org/en/products/mr-beam-pappelsperrholz-bunt"),
+                variant: {
+                    color: {
+                        "e7d27f": "40760806080623"
```

Comment:
risky business but we will make it more solid once we load the material-settings from the new tool. You could add already the rest of the colours and variant options... But since they are hex... we will need to remember to double check them anyway if we ever add them to the frontend... so we can leave it like this if you'd like

Reply:
Yes that's why I left it this way because we need a more solid approach. I think we can change it when implementing the material store links, but anyway this will be redesigned with the new material settings implementation too.